src/types/component-props/layouts/section-with-header.ts

```diff
@@ -7,6 +7,10 @@ export interface SectionWithHeaderProps extends LayoutCommonProps {
     type: ComponentType.Layout;
     descriptor: LayoutType.SectionWithHeader;
     regions: {
+        intro: {
+            components: ComponentProps[];
+            name: 'content';
```

Comment:
For korrekthetens skyld; name vil være 'intro' (name og key'en til en region er alltid det samme)